app/lib/feed_manager.rb

```diff
@@ -26,34 +26,42 @@ def filter?(timeline_type, status, receiver_id)
     end
   end
 
-  def push(timeline_type, account, status)
-    return false unless add_to_feed(timeline_type, account, status)
-
-    trim(timeline_type, account.id)
-
-    PushUpdateWorker.perform_async(account.id, status.id) if push_update_required?(timeline_type, account.id)
-
+  def push_to_home(account, status)
```

Comment:
Is there a good reason to call `push_to_home` and `push_to_list` separately outside of FeedManager? Should we just expose `push_to_user` and let FeedManager sort out lists? (This seems less efficient, but perhaps cleaner. I can see arguments on either side.)

Reply:
As I started writing the code, the push behaviour was different enough that I had to create a separate push_to_list method. But as I refactored, they turned out pretty similar after all. However, I am absolutely OK with keeping them separate for now. As they say, the DRY principle kicks in for n >= 3

Reply:
My thought was that it mirrors push_to_hashtag, which isn't in feed_manager but does need to happen

Reply:
Huh? No, hashtags timelines are not stored in redis

Reply:
Yes, but it mirrors the necessity of calling all of those different functions (see the calling site)